src/openrct2/world/map_animation.c

```diff
@@ -456,7 +456,7 @@ static bool map_animation_invalidate_wall_door(sint32 x, sint32 y, sint32 baseZ)
 	if (gCurrentTicks & 1)
 		return false;
 
-	bool wasInvalidated = false;
+	bool removeAnimation = true;
```

Comment:
Does this detect if it's an animation or does it just play through the animation? If it's the latter, wouldn't it be more appropriate to have it "finishAnimation"? Remove sounds like it makes it static forever. Just my opinion though, what do you think?